VAMobile/src/components/EncourageUpdate.tsx

```diff
@@ -61,6 +61,19 @@ export const EncourageUpdateAlert = () => {
     }
   }, [demoMode])
 
+  useEffect(() => {
+    if (
+      featureEnabled('inAppUpdates') &&
+      storeVersion &&
+      localVersionName &&
+      skippedVersion &&
+      skippedVersion !== storeVersion &&
+      ((isIOS() && storeVersion > localVersionName) || (!isIOS() && +storeVersion > +localVersionName))
+    ) {
```

Comment:
It'd be nice to have this logic stored in a variable since it's used in multiple places and is a pretty beefy conditional

Reply:
updated